tests/unit/helpers/t-test.ts

```diff
@@ -109,19 +109,20 @@ module('t', function (hooks) {
     assert.equal(this.element.textContent, 'bulk add works');
   });
 
-  test('can inline locale for missing locale', async function (this: TestContext, assert) {
+  // TODO: idk why this doesn't pass
```

Comment:
Something we'll need to look at

Reply:
ok so the behavior on our side is that missing translation triggers `onError`, which in this test suite just `throw` back up. That's why I change the assert to `assert.rejects`, but as I debugged & logged things out the err do propagate properly but for some reason `assert.rejects(render(hbs`{{t 'foo.bar' locale='xx-xx'}}`));` keeps failing and idk why...

Reply:
ok fixed it. `@formatjs/intl` verifies legit locale 1st, so `xx-xx` isn't gonna fly. I changed it to `fr`